tests/behavior/transforms/json_parser.toml

```diff
@@ -30,3 +30,19 @@
       type = "check_fields"
       "target_field.a.equals" = 3
       "target_field.b.c.equals" = 4
+
+[transforms.json_parser_null_value]
+  inputs = []
+  type = "json_parser"
+[[tests]]
+  name = "json_parser_null_value"
+  [tests.input]
+    insert_at = "json_parser_null_value"
+    type = "raw"
+    value = "{\"a\": 3, \"b\": null}"
+  [[tests.outputs]]
+    extract_from = "json_parser_null_value"
+    [[tests.outputs.conditions]]
+      type = "check_fields"
+      "a.equals" = 3
+      "b.equals" = "<null>"
```

Comment:
So does this imply that `"<null>"` will be read as a `null` by Vector?

Reply:
No, `"<null>"` will be read as a string (more precisely, `Bytes`) by Vector. So it is just a string representation produced by `to_string_lossy()` and `to_bytes()`. It is not revertible for other data types too, for example Vector timestamps are rendered as strings in ISO 8601 format, but strings containing timestamps in ISO 8601 format are read as strings, not as timestamps.